### ISSUE TITLE
Change .rtlocal.rc settings without rebuilding image?

### ISSUE THREAD

**Issue by ilosh** [crazy-max/docker-rtorrent-rutorrent#216]
Hi,
is there any way to store my own settings in .rtlocal.rc without rebuilding the Docker image?

I changed the settings in /etc/rtorrent/.rtlocal.rc to my preferences over the console but it won't last permanently.

Thank you for your help.

**Comment by HuwSy** [comment]
This could be related to bug 191? As the rtorrent.rc allows edits and still gets loaded in from outside the container.

**Comment by BloodyIron** [comment]
Yeah rather surprising changing this config file wasn't baked into this image :/ it's pretty key to tuning for performance. Especially since things like system.file.allocate.set = 1 aren't even set by the looks of it.
